Design/Workbench/design-layout.md

```diff
@@ -0,0 +1,149 @@
+Summary
+=======
+
+This document describes the layout and high-level details of the new matplotlib-based workbench that will replace MantidPlot.
+
+Workflow
+========
+
+The new workbench will be developed on a long-running feature branch within the main [`mantid`][mantidrepo] repository. The exception to this is the new directory
+structure described below for the graphical components. These changes will be made on the `master` branch prior to any work starting on the workbench. This
+will minimize the effect of merge conflicts on existing components that are going to be reused.
```

Comment:
Probably a day to do the work (by somebody that actually understands `cmake`) and a day for issues that are caused on build servers/developer systems.